inc/vcf/error.hpp

```diff
@@ -36,14 +55,9 @@ namespace ebi
                   message{message} {}
 
 
-        size_t get_line() const
-        {
-          return line;
-        }
-        const std::string &get_raw_message() const
-        {
-          return message;
-        }
+        size_t get_line() const { return line; }
+        const std::string &get_raw_message() const { return message; }
+        virtual ErrorCode get_code() const { return ErrorCode::error; }
```

Comment:
oops, running tests, it seems that it's not so easy. when using the children 2-argument constructor, we would need to add another constructor like 
```
class FileformatError : public MetaSectionError
{
  public:
    // using MetaSectionError::MetaSectionError; // compilation error: constructor with 2 arguments collides
    FileformatError(size_t line) : FileformatError{line, "Error in file format section"} { }
    FileformatError(size_t line, std::string message) : FileformatError{ErrorCode::fileformat, line, message} { }// compilation error: there's no FileformatError that takes 3 arguments
};
```
and removing the `using Error::Error` as we are redefining the 2-argument constructor, but then the 3-argument is not accesible with the same name, nor even with the parent's name (MetaSectionError in this example), and calling a non-direct parent is not allowed, so we can not even `FileformatError(size_t line, std::string message) : Error{ErrorCode::fileformat, line, message} { }`.

Unless I did something very silly, I would let this as it was before.